independent-publisher-connectors/Shopify Admin Content/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2895 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Shopify Admin Content",
+    "description": "The Shopify REST Admin API lets you build apps and other integrations for the Shopify admin.",
+    "contact": {
+      "name": "Hitachi Solutions",
+      "email":"jlindstrom@hitachisolutions.com"
+    }
+  },
+  "host": "thatapiguy.myshopify.com",
+  "basePath": "/admin/api/2021-04/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/blogs/{blog_id}/articles.json": {
+      "get": {
+        "description": "Retrieves a list of all articles from a certain blog",
+        "summary": "Get a list of all articles of a blog",
+        "tags": [
+          "Article"
+        ],
+        "operationId": "ListBlogArticles",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "",
```

Comment:
@thatapiguy Why the description value is set to empty string

Reply:
This header is internal and the user doesn't need to see any description for it. I will add Content Type as the description anyways.